index.js

```diff
@@ -513,7 +513,8 @@ class GithubScm extends Scm {
             }));
         } catch (err) {
             winston.error('Failed to getOpenedPRs: ', err);
-            throw err;
+
+            return [];
```

Comment:
If we are not returning the error, then we can `return [];` in finally block

Reply:
I was wrong about the previous comment. Return in `finally` block always override the return value in `try` block.

Maybe we can write something like 

```javascript
        let pullRequests = [];
        try {
           pullRequests = await this.breaker.runCommand({	            const pullRequests = await this.breaker.runCommand({
                action: 'list',	                action: 'list',
                scopeType: 'pulls',	                scopeType: 'pulls',
                token,	                token,
                params: {	                params: {
                    owner,	                    owner,
                    repo,	                    repo,
                    state: 'open'	                    state: 'open'
                }	                }
            });	            });
            return pullRequests.data.map(pullRequest => ({	            return pullRequests.data.map(pullRequest => ({
                name: `PR-${pullRequest.number}`,	                name: `PR-${pullRequest.number}`,
                ref: `pull/${pullRequest.number}/merge`,	                ref: `pull/${pullRequest.number}/merge`,
                username: pullRequest.user.login,	                username: pullRequest.user.login,
                title: pullRequest.title,	                title: pullRequest.title,
                createTime: pullRequest.created_at,	                createTime: pullRequest.created_at,
                url: pullRequest.html_url,	                url: pullRequest.html_url,
                userProfile: pullRequest.user.html_url	                userProfile: pullRequest.user.html_url
            }));	            }));
        } catch (err) {
            winston.error('Failed to getOpenedPRs: ', err);	            winston.error('Failed to getOpenedPRs: ', err);
            // throw err;
        } finally {
            return pullRequests;
	    }
```

Have a default value. 🤔

Reply:
I think it's better practice not to do `return` in `finally`

getting eslint error `no-unsafe-finally` when I try your suggestion
https://eslint.org/docs/rules/no-unsafe-finally

Reply:
OK. Lets return in catch.